### ISSUE TITLE
Not working on Rewards.GG v2.0

### ISSUE THREAD

**Issue by ghost** [DeathMiner/RewardsGG-Farm#26]
It just says "LOADING THIS SHIT!"

**Comment by demaisj** [comment]
Hi,

Unfortunately since latest rewards.gg update, the script is completely broken. Please disable it to enjoy the new site update.

I'll do an update when I will have the time, unfortunately I'm not available for the whole next month. I still accept Pull Requests, so if someone knows how to code html & js, feel free to contribute to this project.

Hope you'll understand.

**Comment by Xx-HEKZ-xX** [comment]
@DeathMiner I understand what you are going through. I would love to colaborate once I am through with exams(2-3 weeks more). Till then any one who wants to mine can use an auto-clicker( I personally use  this one https://sourceforge.net/projects/orphamielautoclicker/ ) Just set it on 20 minutes and 30 seconds over the get tickets button while you are away... p.s don't do it too much or you might face a ban..  ;)

**Comment by demaisj** [comment]
The problem with the autoclicker is that it is not recognized as a human, because firstly the mouse is not moving, & secondly it is clicking the Get Tickets button at regular intervals. I will do on my side a system to "emulate" human behavior by clicking on the Get Tickets button randomly.

**Comment by Xx-HEKZ-xX** [comment]
As far as i know they do not check for mouse movement but the auto clicker is pretty random but not too random as the site is never in sync with it so sometimes it may click before or after 3-4 seconds. And I have been using this after the update came out for 7 hrs a day and  have mined over 28k. But a real code would work better as it would be more random and you would not have to keep the mouse at one place.

**Comment by LePoissonBleu** [comment]
I did not check their code. But they could be checking click location easily. It is close to impossible to click to same pixel twice as a human just to mention 1 of the things they could do. For me, Im waiting for DeathMiner as I dont wanna risk the ban. Detecting a simple autoclicker is way easier then 1 could imagine.

P.S. Even if they dont check click loc. 1 update to do that and we would get all banned. I doubt DeathMiner would want that.

**Comment by Xx-HEKZ-xX** [comment]
@LePoissonBleu  Yeah nobody wants a ban. I too do not recommend using a auto-clicker but if someone is too eager to get tickets it's their choice and also some people have been soft banned for 4-5 hours because they left the site on auto-click for 1-2 days... from what i have read on the chat.

**Comment by demaisj** [comment]
Because I noticed there was two modes, maybe the script will only work on slow mode, because in this mode it is allowed to be afk, and the script acts like someone that is afk. It might not get the best amount of tickets, but you can always switch off the script to farm in fast mode.

**Comment by LePoissonBleu** [comment]
1 thing is for certain. Part of it can be automated as it make a 'beep' sound when it's ready to be collected. Maybe just need a RNG to make it pick it up between 1-15 sec after it's rdy to be collected.

If the slow mode is what I think it is. Im sure it would be the best bet to stay on the safe course.

**Comment by Zalatis** [comment]
Hey nice script but an update coming soon ? :x

**Comment by SimonNitzsche** [comment]
Btw. It seems that G-Recaptcha resets after 20min so the bot-protection is useless. Its made for fast spam and not for 20min-click.
